goon/code/obj/machinery/bot/chefbot.dm

```diff
@@ -0,0 +1,195 @@
+/obj/machinery/bot/chefbot
+	name = "Chef RAMsay"
+	desc = "(icon, name, concept, and any kind of consistency or sense is currently pending)"
+	icon = 'goon/icons/obj/aibots.dmi'
+	icon_state = "chefbot-idle"
+	density = 1
+	anchored = 0
+	on = 1 // ACTION
+	health = 50
+	var/raging = 0
+	var/list/calledout = list()
+	var/shit_reagents = list(TOXIN,STOXIN,SLIMEJELLY,PLASMA,NANITES,HONKSERUM,SILENCER,DISCOUNT,AMATOXIN,PSILOCYBIN,CARPPHEROMONES,BUSTANUT,ROGAN,MOONROCKS,TOXICWASTE,CHEMICAL_WASTE,HORSEMEAT,OFFCOLORCHEESE,BONEMARROW,IRRADIATEDBEANS,MUTATEDBEANS,MINTTOXIN,MERCURY,MINDBREAKER,SPIRITBREAKER,RADIUM,URANIUM,CARPOTOXIN,ZOMBIEPOWDER,AMUTATIONTOXIN,PACID,CHLORALHYDRATE,LITHIUM,HELL_RAMEN)
```

Comment:
Why no Silencer?
And I know why not minttoxin, but personally I think that should be allowed